src/main/java/uk/gov/cshr/repository/InviteRepositoryMockImpl.java

```diff
@@ -0,0 +1,133 @@
+package uk.gov.cshr.repository;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Repository;
```

Comment:
If this is a mock class should it go under `src/test/java` instead of `src/main/java`?

Reply:
No as its an actual implementation that is used when you run it.  In order to use it you have to run it with the spring profile called "runMocks".  As long as this profile not used in production then we are ok.

It could be put in the test folder if it was used by the unit tests.  But thats not what this is.  Its used by developers and testers to do integration testing locally.